test/controllers/pages_controller_test.rb

```diff
@@ -0,0 +1,10 @@
+require "test_helper"
+
+describe PagesController do
+  describe "index" do
+    it "can get the root path" do
+      get root_path
+      must_respond_with :success
```

Comment:
The spotlight presents us with an interesting edge case when there are no works in the database. Do your controller and view handle that without crashing?